community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/pipes/Pipe.scala

```diff
@@ -99,7 +99,7 @@ case class SingleRowPipe()(implicit val monitor: PipeMonitor) extends Pipe with
   def symbols: SymbolTable = new SymbolTable()
 
   def internalCreateResults(state: QueryState) =
-      Iterator(ExecutionContext.empty)
+    Iterator(state.initialContext.getOrElse(ExecutionContext.empty))
```

Comment:
Is this a violation of `SingleRow`?  Basically we want to be able to have a `SingleRow` on the right-hand-side of an `Apply`. We could also create a new operator for this or use an empty `Argument`, but this doesn't seem to break anything so we opted for modifying `SingleRow` instead. Opinions? @systay ?